po/tr.po

```diff
@@ -1065,9 +1061,8 @@ msgstr "Gerekli araç kullanılamıyor"
 
 #: src/tools/ui/tool_arc.ui:5 src/tools/ui/tool_line.ui:5
 #: src/tools/ui/tool_pencil.ui:5
-#, fuzzy
 msgid "Line shape"
-msgstr "Fırça formu"
+msgstr "Çizgi Şekli"
```

Comment:
Fırça şekli

Reply:
ok i'll change that myself, no need to do a new pull request